docs/data/indexers/build-your-own/ingest-sdk/developer_guide/ledgerbackends/captivecore.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Captive Core
+sidebar_position: 0
+---
+
+Captive Core invokes the `stellar-core` binary as a subprocess to stream ledgers from the Stellar network. It can be used to stream a ledger range from the past or to stream new ledgers whenever they are confirmed by the network.
+
+## Prerequisites
+
+Using captive Core requires stellar-core binary to be installed. Since stellar-core is resource-intensive, it has specific hardware requirements outlined here: [Hardware Requirements](/docs/validators/admin-guide/prerequisites#hardware-requirements).
```

Comment:
```suggestion
Using captive Core requires stellar-core binary to be [installed](/docs/validators/admin-guide/installation) first. 
```
This would avoid repeating details of core aspects. And consider deleting the `Installation` section which follows here, lines 12 - 23, as the core admin guide referral addresses this aspect also.